Example NestJS app using otel lib to send Traces to CodeSee (#30)
Codesee-io/examples: nestjs-otel-lib-example/README.md

# NestJS integration with OpenTelemetry library + CodeSee example

A sample Typescript application using the otel lib to send Traces data to CodeSee.

> ## How Traces from sample app get into CodeSee?

Reviewer: ```suggestion
## Adding CodeSee to your existing Nest.js app
```

[tracing.ts](src/tracing.ts): It loads in required otel packages to be able to send Traces data to Codesee.

> [main.ts](src/main.ts): Load in the `tracking.ts` and starts it. Make sure to import tracing module before any other modules.

Reviewer: ```suggestion
[main.ts](src/main.ts): Loads in the `tracing.ts` and starts it. Make sure to import tracing module before any other modules.
```

[env-example](env-example): Copy this file as `.env` and update its content to fill in required CodeSee token.

> ## Copy .env file

Reviewer: ```suggestion
# Running this application
## Copy .env file
```
